src/osdc/Filer.h

```diff
@@ -43,12 +45,74 @@ class OSDMap;
 class Finisher;
 
 
+struct C_TruncRange;
+
+class TruncWorkQueue : public md_config_obs_t {
```

Comment:
Please reorganize classes to follow the project guidelines: https://google.github.io/styleguide/cppguide.html#Declaration_Order

In particular, public members hsould be declared first.

Reply:
I tried to reorganize `TruncWorkQueue` and `C_OnTruncWorkQueue` and hope I did it right.
Sorry that I'm not familiar with google style guidelines.

Reply:
Much better, thanks!